generators/client/templates/react/package.json.ejs

```diff
@@ -123,7 +123,7 @@ limitations under the License.
     "jest-sonar-reporter": "2.0.0",
     "json-loader": "0.5.7",
     <%_ if (!skipCommitHook) { _%>
-    "lint-staged": "9.5.0",
+    "lint-staged": "8.2.1",
```

Comment:
Can you add a template comment with link to #11131 to explain that we don't use the latest version because of issues on Windows. Also, can you add it for Angular as well.

Reply:
Good idea! I added comments.